database/migrations/2021_03_04_095453_add_foreign_keys.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddForeignKeys extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('open_graph_scans', function (Blueprint $table) {
+            $table->bigInteger('website_id')->unsigned()->index()->change();
```

Comment:
Thanks for this, much appreciated - Do we know 100% that changing this from `increment` to `bigInt` won't adjust the autoinc on the db?

Reply:
I verified it.
Here is the output of "SHOW CREATE TABLE open_graph_scans" after the migration got applied:

```
MariaDB [odin]> show create table open_graph_scans \G;
*************************** 1. row ***************************
       Table: open_graph_scans
Create Table: CREATE TABLE `open_graph_scans` (
  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `website_id` bigint(20) unsigned NOT NULL,
  `icon` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `title` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `description` text COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `url` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `image` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `created_at` timestamp NULL DEFAULT NULL,
  `updated_at` timestamp NULL DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `open_graph_scans_website_id_index` (`website_id`),
  CONSTRAINT `open_graph_scans_website_id_foreign` FOREIGN KEY (`website_id`) REFERENCES `websites` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci
```

But i agree it would be better to use bigIncrements here.